backend/corpora/lambdas/api/v1/curation/collection_uuid/__init__.py

```diff
@@ -13,14 +13,13 @@ def delete_collection(collection_uuid: str, token_info: dict):
     collection = get_collection(
         db_session,
         collection_uuid,
-        visibility=CollectionVisibility.PRIVATE,
         owner=owner_or_allowed(token_info),
         include_tombstones=True,
     )
-    if collection.revision_of:
-        raise MethodNotAllowedException("Can only delete a private collection through curation API.")
-    elif collection.tombstone:
+    if collection.tombstone:
```

Comment:
If the collection is tombstoned, that means it _was_ a public Collection, in which case there should still be an error response, not 204.

As the already-deleted private collection returns 204, the already-deleted public collection should return an error.

Reply:
So I think you should just remove this line, and change the `elif` on ln 21 to `if`

Reply:
And obviously update the test below accordingly